app/src/main/java/org/fossasia/susi/ai/login/LoginPresenter.kt

```diff
@@ -1,85 +1,132 @@
 package org.fossasia.susi.ai.login
 
 import android.content.Context
+import android.util.Patterns
```

Comment:
I found [here](https://medium.com/@cervonefrancesco/model-view-presenter-android-guidelines-94970b430ddf) that presenter doesn't depend on android classes.